native_client/deepspeech.h

```diff
@@ -48,6 +48,16 @@ typedef struct Metadata {
   double confidence;
 } Metadata;
 
+/**
+ * @brief Stores Metadata structs for each alternative transcription
```

Comment:
This is not an opaque data structure, so it's going to need proper documentation for all bindings.